src/styles/utils/_variables.scss

```
@@ -0,0 +1,11 @@
+$day-size: 100px;
+$font-family: arial, sans-serif;
+$font-size: 30px;
+$days:
+  "mon" 0px,
+  "tue" 101px,
+  "wed" 202px,
+  "thu" 303px,
+  "fri" 404px,
+  "sat" 505px,
+  "sun" 606px;
```

Hard-coded values again. Changes pixels to indexes and then you will be able to calculate the margin basing on that ;)
```suggestion
$days:
  "mon" 0px,
  "tue" 1,
  "wed" 2,
  "thu" 3,
etc...
```